src/phone-number-input/phone-number-input.styles.tsx

```diff
@@ -0,0 +1,109 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { Icon } from "../icon/icon";
+import {
+    ElementBoundary as InputSelectElemBoundary,
+    Wrapper as InputSelectWrapper,
+} from "../input-select/input-select.styles";
+import { Text } from "../text/text";
+import { TextStyle } from "../text/text-style";
+import { Transition } from "../transition";
+
+interface StyleProps {
+    disabled?: boolean;
+    $expanded?: boolean;
+}
+
+// =============================================================================
+// STYLING
```

Comment:
These styles look exactly the same as the `input-group-list-addon.style.tsx`. May i suggest moving that to `shared` instead and reuse them.

Reply:
alternatively, use from input group